templates/customers/register.liquid

```diff
@@ -110,7 +117,8 @@
         <svg aria-hidden="true" focusable="false" role="presentation">
           <use href="#icon-error" />
         </svg>
-        {{ form.errors.translated_fields['password'] | capitalize }} {{ form.errors.messages['password'] }}.
+        {{ form.errors.translated_fields['password'] | capitalize }}
+        {{ form.errors.messages['password'] }}.
       </span>
```

Comment:
I think we need some funky looking placement here cause we're getting some whitespace issue. Which we were getting prior to the reformatting. 
![](https://screenshot.click/19-08-6kbto-3nsm0.png)

Reply:
Same thing here. The _source_ had whitespace in it so we understood that it was meaningful and thus didn't change the output. I think it's working as intended. If the source _lacked_ whitespace here, it would have added a whitespace stripping character to the error message drop.